src/geode/model/helpers/convert_to_mesh.cpp

```diff
@@ -429,6 +429,23 @@ namespace geode
         auto mesh = geode::detail::create_mesh< SolidMesh3D >( meshes );
         auto mesh_builder = geode::SolidMeshBuilder< 3 >::create( *mesh );
         ModelToMeshMappings brep2mesh;
+        for( const auto unique_vertex :
+            geode::Range{ brep.nb_unique_vertices() } )
+        {
+            bool is_in_block{ false };
+            for( const auto& component :
+                brep.component_mesh_vertices( unique_vertex ) )
+            {
+                if( component.component_id.type()
+                    == geode::Block3D::component_type_static() )
+                {
+                    is_in_block = true;
+                    break;
+                }
+            }
```

Comment:
you can use has_component_mesh_vertices(unique_vertex,geode::Block3D::component_type_static() ), it is simpler